addons/mail/static/src/discuss/core/common/discuss_notification_settings.xml

```diff
@@ -4,6 +4,14 @@
     <t t-name="mail.DiscussNotificationSettings">
         <div class="o-mail-DiscussNotificationSettings d-flex flex-column">
             <div class="d-flex flex-column my-1">
+                <label class="cursor-pointer d-flex align-items-center my-1">
+                    <h5>Message sound</h5>
+                    <div class="flex-grow-1"/>
+                    <div class="form-check form-switch">
+                        <input class="form-check-input" type="checkbox" role="switch" t-att-checked="store.settings.messageSound" t-on-change="onChangeMessageSound"/>
+                    </div>
+                </label>
```

Comment:
Maybe move it last as this is a set once and forget option and a rather minor one, as opposed to for example "mute all" which is possibly more frequent.

Reply:
Yes, I would think eventually we'd have a more complete "sound" category.